src/binfhe/include/binfhe-base-scheme.h

```diff
@@ -103,6 +103,19 @@ class BinFHEScheme {
     LWECiphertext EvalBinGate(const std::shared_ptr<BinFHECryptoParams>& params, BINGATE gate, const RingGSWBTKey& EK,
                               ConstLWECiphertext& ct1, ConstLWECiphertext& ct2) const;
 
+    /**
+   * Evaluates a binary gate on a vector of ciphertexts (calls bootstrapping as a subroutine)
+   *
+   * @param params a shared pointer to RingGSW scheme parameters
+   * @param gate the gate; can be AND, OR, NAND, NOR, XOR, or XOR
```

Comment:
Update the doxygen comment; I would list relevant gates and add some notes 3- and 4-input gates